src/components/Input/Input.tsx

```diff
@@ -85,15 +85,15 @@ class Input extends UIComponent<any, any> {
     const { children, className, icon, input, type } = this.props
     const [htmlInputProps, restProps] = this.partitionProps()
 
-    const inputClasses = cx(classes.input)
+    const inputClasses = (cx as any)(classes.input)
```

Comment:
Can we `yarn add --dev @types/classnames` to avoid this?

Reply:
we do have it but it still throws

Reply:
reverted the change